### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by dxp10** [yeonjulee1005/yeonjulee1005#4]
Hi friend, could you rate my repositories and I'll rate yours at the same time? Thanks!

**Comment by yeonjulee1005** [comment]
of course!!:) i’ll rate up!나의 iPhone에서 보냄2023. 3. 2. 오후 6:20, dxp_10 ***@***.***> 작성:﻿
Hi friend, could you rate my repositories and I'll rate yours at the same time? Thanks!

**Comment by dxp10** [comment]
Thank you, I appreciate your projects too
